src/onyx/messaging/aeron.clj

```diff
@@ -17,6 +17,38 @@
              [java.util.function Consumer]
              [java.util.concurrent TimeUnit]))
 
+(def max-port 50000)
```

Comment:
That's a good point - and most machines won't honestly be able to run more than a few dozen peers before having CPU lock up. Let's leave it as is.